### ISSUE TITLE
Menu is not defined

### ISSUE THREAD

**Issue by Yadro** [bokuweb/react-motion-menu#31]
Error: `index.js:174 Uncaught ReferenceError: Menu is not defined`

After rename Menu to MotionMenu
`index.js:133 Uncaught TypeError: Cannot read property 'props' of undefined`

**Comment by Yadro** [comment]
if `this.props.children` have one component then `_toArray(this.props.children)` return undefined (index.js:174)

**Comment by Yadro** [comment]
And last error )
```js
stripStyle.js:16 Uncaught TypeError: Cannot read property 'val' of undefined
    at Object.stripStyle [as default] (stripStyle.js:16)
    at Object.getInitialState (Motion.js:55)
    at new <anonymous> (ReactClass.js:652)
    at ReactCompositeComponent.js:295
    at measureLifeCyclePerf (ReactCompositeComponent.js:75)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (ReactCompositeComponent.js:294)
    at ReactCompositeComponentWrapper._constructComponent (ReactCompositeComponent.js:280)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:188)
    at Object.mountComponent (ReactReconciler.js:46)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:371)
```

**Comment by bokuweb** [comment]
@Yadro Sorry, it is fixed @v0.3.1. pleaase re-install `react-motion-menu`.
